web-ui/modules/i18n.dict.mjs

```
@@ -3197,7 +3197,1063 @@ const DICT = Object.freeze({
+        'common.refresh': 'Làm mới',
+        'common.refreshing': 'Đang làm mới...',
+        'common.loading': 'Đang tải...',
+        'common.saving': 'Đang lưu...',
+        'common.sending': 'Đang gửi...',
+        'common.scanning': 'Đang quét...',
+        'common.export': 'Xuất',
+        'common.import': 'Nhập',
+        'common.apply': 'Áp dụng',
+        'common.applying': 'Đang áp dụng...',
+        'common.confirming': 'Đang xác nhận...',
+        'common.writeToEditor': 'Ghi vào trình soạn thảo',
```

Biome (2.4.16)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)